bse/bsesong.cc

```diff
@@ -991,6 +967,36 @@ SongImpl::loop_right (int tick)
     }
 }
 
+int
+SongImpl::tick_pointer() const
+{
+  BseSong *self = const_cast<SongImpl*> (this)->as<BseSong*>();
+
+  return self->tick_SL;
+}
+
+void
+SongImpl::tick_pointer (int tick)
+{
+  BseSong *self = as<BseSong*>();
+
+  if (uint (tick) != self->tick_SL)
+    {
+      // this property has no undo
+
+      BSE_SEQUENCER_LOCK ();
+      self->tick_SL = tick;
+      for (SfiRing *ring = self->tracks_SL; ring; ring = sfi_ring_walk (ring, self->tracks_SL))
+        {
+          BseTrack *track = (BseTrack*) ring->data;
+          track->track_done_SL = FALSE;	/* let sequencer recheck if playing */
```

Comment:
While you're at it, the C macros TRUE and FALSE should be converted to C++'s true and false.